docker-compose.yml

```
@@ -0,0 +1,75 @@
+#Developpement docker-compose file by Uniride
+version: '3.8'
+
+services:
+
+#This container is used to get the frontend
+  uniride_frontend:
+    container_name: uniride_frontend
+    image: uniride1/uniride_front #automatically latest
```

```suggestion
    image: uniride1/uniride_front
```
On est censé le savoir